support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -403,7 +403,7 @@ Resources:
           ReturnData: false
       ComparisonOperator: LessThanOrEqualToThreshold
       Threshold: 0
-      EvaluationPeriods: 36
```

Comment:
having reviewed the stats, I think 2h should be ample to avoid false alarms.  I reviewed all the rest including contributions and they all look ok.
There may even be scope for a no digital subs via Direct debit / paypal/ stripe separately.